ts/src/phemex.ts

```diff
@@ -589,8 +589,11 @@ export default class phemex extends Exchange {
         //
```

Comment:
Hum I can't create orders 
```
 p phemex createOrder "LTC/USDT" market buy 0.1 --sandbox
Python v3.10.9
CCXT v3.0.38
phemex.createOrder(LTC/USDT,market,buy,0.1)
Traceback (most recent call last):
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 232, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 211, in main
    result = await method(*args)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/phemex.py", line 2168, in create_order
    request['baseQtyEv'] = self.to_ev(amountString, market)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/phemex.py", line 1011, in to_ev
    return self.to_en(amount, market['valueScale'])
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/phemex.py", line 1003, in to_en
    stringValue = str(precise)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/base/precise.py", line 173, in __str__
    integer_array = list(str(abs(self.integer)).rjust(self.decimals, '0'))
TypeError: 'float' object cannot be interpreted as an integer
```

Reply:
@ndubel I will, but when we edit something it is essential to test on Python as well because, as you know the languages are not 100% equivalent 🙂

Reply:
still not working, but I will try to figure it out 
```
p phemex createOrder "ANKR/USDT" market buy 100 --sandbox
Python v3.10.9
CCXT v3.0.38
phemex.createOrder(ANKR/USDT,market,buy,100)
Traceback (most recent call last):
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 232, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 211, in main
    result = await method(*args)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/phemex.py", line 2168, in create_order
    request['baseQtyEv'] = self.to_ev(amountString, market)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/phemex.py", line 1011, in to_ev
    return self.to_en(amount, market['valueScale'])
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/phemex.py", line 1006, in to_en
    return int(floatString)
ValueError: invalid literal for int() with base 10: '1000000.0'
```